README.md

```diff
@@ -180,9 +187,9 @@ The following is an example of what a config file might look like:
         ]
     },
     "flag": {
-        "field": "STATUS",
```

Comment:
@allisonmuise The reason we changed it is so that we can store the Service Request status from Cityworks in the STATUS field on the layer.